### PR TITLE
WHL scale testing - Text Logs

### PR DESCRIPTION
**Pull request by chris-raitano**
## Summary

This PR creates a container for generating text logs for scale testing. 

This builds off of PR #1066 _WHL Scale Testing - Log GenerationInfra_. Reviewers should view the [diff](https://github.com/microsoft/Docker-Provider/compare/craitano/log-generation-infra...craitano/text-log-generation) between these branches to see the changes made by this PR 

### Type of change:

- [ ] Feature work
- [ ] Bugfix
- [ ] Refactoring
- [ ] Documentation
- [ ] Developer Experience
- [x] Test
- [ ] Logging/Metrics
- [x] Automation 

## Testing

- [ ] Unit tests
- [ ] Integration tests
- [x] Manual tests (add details)

Follow the instructions in Section 5 of [whl-scale-test.md](https://github.com/microsoft/Docker-Provider/blob/craitano/text-log-generation/test/whl-scale-tests/whl-scale-test.md#5-generating-logs-for-testing) to deploy the text logs generator.

### CODE REVIEW

File: test/whl-scale-tests/geneva-config-files/text_log_geneva_config.xml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+          <MdmDimension name="CloudTenant"><![CDATA[GetEnvironmentVariable("AKS_CLUSTER_NAME")]]></MdmDimension>
+          <MdmDimension name="CloudRole"><![CDATA[GetEnvironmentVariable("HOSTNAME")]]></MdmDimension>
+          <MdmDimension name="RoleInstance"><![CDATA[GetEnvironmentVariable("AKS_REGION")]]></MdmDimension>
+          <MdmDimension name="Environment"><![CDATA[GetEnvironmentVariable("MONITORING_ENVIRONMENT")]]></MdmDimension>
+          <MdmDimension name="Datacenter"><![CDATA[GetEnvironmentVariable("MONITORING_DATACENTER")]]></MdmDimension>
+          <MdmDimension name="ScaleUnit"><![CDATA[GetEnvironmentVariable("MONITORING_TENANT_NAME")]]></MdmDimension>
+        </MdmDimensions>
+      </CounterSet>
+    </CounterSets>
+    <TextLogSubscriptions>
```
Comment (cacarlt) [inline test/whl-scale-tests/geneva-config-files/text_log_geneva_config.xml:66]: We should make a note here that this won't use the Managed Fluent feature

File: test/whl-scale-tests/whl-scale-test.md
Comment (cacarlt) [inline test/whl-scale-tests/whl-scale-test.md:1]: Make sure to add new script and config maps usage to the markdown file. If there are changes represented in XML that use the same value in the configmap that should be automated.

**Review by cacarlt — CHANGES_REQUESTED** [review]

File: test/whl-scale-tests/geneva-config-files/text_log_geneva_config.xml (hunk 1 above)
Comment (chris-raitano) [reply test/whl-scale-tests/geneva-config-files/text_log_geneva_config.xml:66]: Added example xml for managed fluent here. Also documented how to switch to managed fluent in the md

Comment (cacarlt) [reply test/whl-scale-tests/geneva-config-files/text_log_geneva_config.xml:66]: Looks good to me

**Review by cacarlt — APPROVED** [review]
Approved

**Pull request merged**